_vendors/gitbook.yaml

```diff
@@ -0,0 +1,12 @@
+---
+base_pricing: $8 per u/m
+footnotes: '[^gitbook-price]: Google SSO is provided for free, but
+  SAML is only included at the Pro tier. The move from Plus to Pro also
+  adds a fixed $99/mo "platform fee". The percent increase reflects this
+  platform fee with a single user.'
+name: GitBook
+percent_increase: 1325%
```

Comment:
Including the flat platform rate as part of the per user cost increase isn't accurate. Please update with the correct percentage and the SSO price to just the per-user price and keep the platform fee in the footnote

Reply:
Done